src/DotNetBoilerplate.Application/Reviews/Create/CreateReviewHandler.cs

```diff
@@ -16,9 +16,9 @@ IUserRepository userRepository
 {
     public async Task<Guid> HandleAsync(CreateReviewCommand command)
     {
-        bool userAlreadyGaveReview = await reviewRepository.UserAlreadyGaveReviewToThisBook(command.BookId, context.Identity.Id);
+        bool userAlreadyGaveReview = await reviewRepository.UserAlreadyGaveReviewToThisBook(command.BookId.Value, context.Identity.Id);
```

Comment:
No i tutaj jak nie będziemy mieli tego nulla to nie musimy na siłę określać że ma to wartość != null